src/models/UserMapper.php

```diff
@@ -38,6 +38,7 @@ public function getVerboseFields()
             "username"         => "username",
             "full_name"        => "full_name",
             "twitter_username" => "twitter_username",
+            "trusted"          => "trusted"
```

Comment:
Sounds like a good idea. I created Issue https://joindin.jira.com/browse/JOINDIN-734 to follow up on that